troubleshooting/known_issues/parry_peak_console_boot_errors.md

```diff
@@ -0,0 +1,77 @@
+# Parry Peak Boot Issue with Console Parameter
```

Comment:
Parry Peak is an internal name. We should update all instances in this PR with the customer facing name: HPE Cray EX255a

Reply:
Noted and updated